app/livechat/server/hooks/sendToCRM.js

```diff
@@ -83,6 +123,57 @@ callbacks.add('livechat.closeRoom', (room) => {
 	return sendToCRM('LivechatSession', room);
 }, callbacks.priority.MEDIUM, 'livechat-send-crm-close-room');
 
+callbacks.add('livechat.newRoom', (room) => {
+	if (!settings.get('Livechat_webhook_on_start')) {
+		return room;
+	}
+
+	return sendToCRM('LivechatSessionStart', room);
+}, callbacks.priority.MEDIUM, 'livechat-send-crm-start-room');
+
+callbacks.add('livechat.afterTakeInquiry', (inquiry) => {
+	if (!settings.get('Livechat_webhook_on_chat_taken')) {
+		return inquiry;
+	}
+
+	const { rid } = inquiry;
+	const room = LivechatRooms.findOneById(rid);
+
+	return sendToCRM('LivechatSessionTaken', room);
+}, callbacks.priority.MEDIUM, 'livechat-send-crm-room-taken');
+
+callbacks.add('livechat.chatQueued', (room) => {
+	if (!settings.get('Livechat_webhook_on_chat_queued')) {
+		return room;
+	}
+
+	return sendToCRM('LivechatSessionQueued', room);
+}, callbacks.priority.MEDIUM, 'livechat-send-crm-room-queued');
+
+callbacks.add('livechat.afterForwardChatToAgent', (params) => {
+	const { rid, oldServedBy } = params;
+	if (!settings.get('Livechat_webhook_on_forward')) {
+		return params;
+	}
+
+	const originalRoom = LivechatRooms.findOneById(rid);
+	const room = Object.assign(originalRoom, { oldServedBy });
+	sendToCRM('LivechatSessionForwarded', room);
+	return params;
+}, callbacks.priority.MEDIUM, 'livechat-send-crm-room-forwarded');
+
+callbacks.add('livechat.afterForwardChatToDepartment', (params) => {
+	const { rid, oldDepartmentId } = params;
+	if (!settings.get('Livechat_webhook_on_forward')) {
+		return params;
+	}
+
+	const originalRoom = LivechatRooms.findOneById(rid);
+	const room = Object.assign(originalRoom, { oldDepartmentId });
+	sendToCRM('LivechatSessionForwarded', room);
+	return params;
+}, callbacks.priority.MEDIUM, 'livechat-send-crm-room-forwarded');
```

Comment:
Should we rename this event? Because it is the same as above or is it correct?

Reply:
Done.